.babelrc

```diff
@@ -1,9 +1,17 @@
 {
-  "presets": ["es2015"],
-  "plugins": [
-    "transform-flow-strip-types",
-    "transform-es3-property-literals",
-    "transform-es3-member-expression-literals",
-    ["transform-es2015-for-of", {"loose": true}]
```

Comment:
We need this `loose` flag, otherwise `for ... of` loops require a `Symbol` polyfill to be loaded in the browser.